### ISSUE TITLE
WebLN support

### ISSUE THREAD

**Issue by diskbuster** [mdance/lnauth#2]
The module works great and provides a fantastic new way to handle Drupal user accounts! 

For a smart user authentication management getalby.com makes a good job, but I didn't get how to add WebLN support in the module to support an easy login by an browser extension.

Maybe you find it useful and have an idea how to get it in?

**Comment by mdance** [comment]
I think it would probably make more sense to create a seperate webln module so its decoupled and generic from this functionality.

**Comment by mdance** [comment]
I just read about the webln standard at https://www.webln.guide it seems pretty easy to implement

**Comment by diskbuster** [comment]
yeah, there is a good guide, but I have no idea for now, how to get it in to have the behavior like here https://webln.twentyuno.net/login
